sdk/python/README.md

```diff
@@ -1,11 +1,74 @@
 # Dagger Python SDK
 
-Install [hatch](https://hatch.pypa.io/latest/install/). Recommend [`pipx`](https://github.com/pypa/pipx)), e.g. `pipx install hatch`
+A client package for running [Dagger](https://dagger.io/) pipelines.
 
-Run tests with `hatch run test`. Assumes `go run ./cmd/cloak dev --workdir sdk/python` is running.
```

Comment:
`hatch run test` will fail without this. I would only remove it when that is no longer the case.

Reply:
I have a tangle of PRs, the fix to that is in https://github.com/dagger/dagger/pull/3716, I'd rather just push this on.